tests/acceptance/features/apiSpaces/trashBin.feature

```diff
@@ -74,11 +72,44 @@ Feature: Restore files, folder
       | Brian | viewer  | 403  | should not           | should               |
 
 
-  Scenario: user can restore a file even if there is not enough quota to do so
-    Given user "Admin" has changed the quota of the "Brian Murphy" space to "30"
-    And user "Brian" has uploaded file with content "file is less than 30 bytes" to "/file.txt"
-    And user "Brian" has uploaded file with content "reduceContent" to "/file.txt"
-    And user "Brian" has uploaded file with content "some content" to "newFile.txt"
-    When user "Brian" restores version index "1" of file "/file.txt" using the WebDAV API
-    Then the HTTP status code should be "204"
-    And the content of file "/file.txt" for user "Brian" should be "file is less than 30 bytes"
+  Scenario Outline: only space manager can purge the trash via the webDav API
+    Given user "Alice" has shared a space "restore objects" with settings:
+      | shareWith | Brian  |
+      | role      | <role> |
+    And the administrator has assigned the role "Space Admin" to user "Brian" using the Graph API
+    And user "Alice" has removed the file "newFolder/file.txt" from space "restore objects"
+    When user "<user>" deletes the file "file.txt" from the trash of the space "restore objects"
+    Then the HTTP status code should be "<code>"
+    And as "<user>" file "file.txt" <shouldOrNotBeInTrash> exist in the trashbin of the space "restore objects"
+    Examples:
+      | user  | role    | code | shouldOrNotBeInTrash |
+      | Brian | manager | 204  | should not           |
+      | Brian | editor  | 403  | should               |
+      | Brian | viewer  | 403  | should               |
+
+
+  Scenario Outline: admin non member of space cannot see the trash bin via the webDav API
+    Given user "Alice" has removed the file "newFolder/file.txt" from space "restore objects"
+    And the administrator has assigned the role "<systemRole>" to user "Brian" using the Graph API
+    When user "Brian" with admin permission lists all deleted files in the trash bin of the space "restore objects"
```

Comment:
this way we can reuse this step later for other use cases
```suggestion
    When user "Brian" lists all deleted files in the trash bin of the space "restore objects"
```

Reply:
we have yet this step `user "Brian" lists all deleted files in the trash bin of the space "restore objects"`
difference between this steps, that user admin can get space if he is not member of the space (using `graph/v1.0/drives/` instead of `graph/v1.0/me/drives/`)

Reply:
So here, we need to get the space id owned by another user and then list its trasbin. Go it